### DISCUSSION TITLE
Jira Github Integration - Add URL to deployments visible in jira

### DISCUSSION THREAD

**Discussion by kukrejaravi9** [atlassian/github-for-jira#2633] [General]
I have my github repository integrated to jira github-for-jira: https://github.com/atlassian/github-for-jira. 

I have github action workflow in my repository which deploys my code and I see those deployments in my jira tickets. Each deployment I do via the workflow has a unique url which I want to provide to my QA team for testing. Is there a way I can add my URL to the deployments visible in jira?

**Comment by devin-purple** [comment]
This would be very useful - there's a way to get to it, but it requires quite a few clicks (Click into deployments, click the deployment, go back to the deployment summary, click the URL in the job).
